micrometer-core/src/test/java/io/micrometer/core/instrument/binder/ExecutorServiceMetricsTest.java

```diff
@@ -38,6 +40,28 @@ void before() {
         registry = new SimpleMeterRegistry();
     }
 
+    @DisplayName("Normal executor can be instrumented after being initialized")
+    @Test
+    void executor() throws InterruptedException {
+        CountDownLatch lock = new CountDownLatch(1);
+        Executor exec = r -> {
+            r.run();
+            lock.countDown();
+        };
+        Executor executor = ExecutorServiceMetrics.monitor(registry, exec, "exec");
+        executor.execute(() -> System.out.println("hello"));
+        lock.await();
+        assertThat(registry.find("exec").timer()).map(Timer::count).hasValue(1L);
+    }
+
+    @DisplayName("thread pool executor can be instrumented even treat as Executor")
```

Comment:
What does `even treat` mean?

Reply:
ah, assign to Executor, so overloading with Executor's one